src/components/viesti/viesti-api.js

```diff
@@ -13,17 +13,24 @@ const url = {
   ketju: id => `${url.ketjut}/${id}`,
   viestit: id => `${url.ketju(id)}/viestit`,
   kasittelijat: '/api/private/kasittelijat',
-  energiatodistusKetjut: id => `/api/private/energiatodistukset/all/${id}/viestit`,
+  energiatodistusKetjut: id =>
+    `/api/private/energiatodistukset/all/${id}/viestit`
 };
 
 export const serialize = R.evolve({
+  'has-kasittelija': Maybe.orSome(null),
+  'include-kasitelty': Maybe.orSome(null),
+  'kasittelija-id': Maybe.orSome(null),
```

Comment:
miksi nämä on tässä?

Reply:
nämä on siis hakuparametreja - ketjulla ei ole näitä tietoja

Reply:
joo, noita ei olis pitänyt olla siellä ja nyt on poistettu